homeassistant/components/roborock/switch.py

```diff
@@ -50,7 +50,18 @@ class RoborockSwitchDescription(
         translation_key="child_lock",
         icon="mdi:account-lock",
         entity_category=EntityCategory.CONFIG,
-    )
+    ),
+    RoborockSwitchDescription(
+        set_command=lambda entity, value: entity.send(
+            RoborockCommand.SET_FLOW_LED_STATUS, {"status": 1 if value else 0}
+        ),
+        get_value=lambda data: data.send(RoborockCommand.GET_FLOW_LED_STATUS),
+        evaluate_value=lambda data: data["status"] == 1,
+        key="status_indicator",
+        translation_key="status_indicator",
+        icon="mdi:alarm-light-outline",
```

Comment:
This is a light... should it have been a light entity?

Reply:
> Right, yet we give it the name "light" and give it a light icon 😬

haha fair enough

> so I guess it is correct as is right now.

Sounds good!

> Sorry for the noise 👍

No apology needed!